packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -23,6 +23,25 @@ describe('makeRecorderApi', () => {
   let rumInit: (options?: { worker?: DeflateWorker }) => void
   let startSessionReplayRecordingManually: boolean
 
+  function setupRecorderApi(sessionManager: RumSessionManager = createRumSessionManagerMock().setId('1234')) {
+    lifeCycle = new LifeCycle()
+    stopRecordingSpy = jasmine.createSpy('stopRecording')
+    startRecordingSpy = jasmine.createSpy('startRecording').and.callFake(() => ({
+      stop: stopRecordingSpy,
+    }))
+
+    recorderApi = makeRecorderApi(startRecordingSpy, createDeflateWorkerSpy)
+    rumInit = ({ worker } = {}) => {
+      recorderApi.onRumStart(
+        lifeCycle,
+        mockRumConfiguration({ startSessionReplayRecordingManually }),
+        sessionManager,
+        {} as ViewContexts,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
        mockViewContexts(),
```